<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(242, 242, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QComboBox" name="comboBox_ScreeNumber">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #ffffff; /* White background */
    color: #333333; /* Dark gray text */
    border: 2px solid #719fee; /* Blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px 10px; /* Padding inside the combobox */
    font-size: 16px; /* Font size */
}

QComboBox:hover {
    background-color: #e6f0ff; /* Light blue background on hover */
    border: 2px solid #5b8bcc; /* Darker blue border on hover */
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px; /* Width of the drop-down button */
    border-left: 1px solid #719fee; /* Border for the drop-down button */
}

QComboBox::down-arrow {
    image: url(:/downArrow.png); /* Custom down arrow icon */
    width: 10px; /* Width of the arrow */
    height: 10px; /* Height of the arrow */
}

QComboBox QAbstractItemView {
    background-color: #ffffff; /* Background of the drop-down list */
    color: #333333; /* Text color in the drop-down list */
    border: 1px solid #719fee; /* Border of the drop-down list */
    selection-background-color: #719fee; /* Background color of selected item */
    selection-color: #ffffff; /* Text color of selected item */
}
</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>551</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
QLabel.title {
    color: #719fee; /* Blue text */
    font-size: 24px; /* Larger font size */
    font-weight: bold; /* Bold text */
    padding: 5px 0; /* Padding */
}

</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;HIGHSCORE SYSTEM CONFIG&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;

QLabel {
    color: #333333; /* Dark gray text */
    font-size: 14px; /* Font size */
    padding: 2px; /* Padding */
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Type of resolution&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Resolution">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #ffffff; /* White background */
    color: #333333; /* Dark gray text */
    border: 2px solid #719fee; /* Blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px 10px; /* Padding inside the combobox */
    font-size: 16px; /* Font size */
}

QComboBox:hover {
    background-color: #e6f0ff; /* Light blue background on hover */
    border: 2px solid #5b8bcc; /* Darker blue border on hover */
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px; /* Width of the drop-down button */
    border-left: 1px solid #719fee; /* Border for the drop-down button */
}

QComboBox::down-arrow {
    image: url(:/downArrow.png); /* Custom down arrow icon */
    width: 10px; /* Width of the arrow */
    height: 10px; /* Height of the arrow */
}

QComboBox QAbstractItemView {
    background-color: #ffffff; /* Background of the drop-down list */
    color: #333333; /* Text color in the drop-down list */
    border: 1px solid #719fee; /* Border of the drop-down list */
    selection-background-color: #719fee; /* Background color of selected item */
    selection-color: #ffffff; /* Text color of selected item */
}
</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1280 x 720</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1920 x 1080</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;

QLabel {
    color: #333333; /* Dark gray text */
    font-size: 14px; /* Font size */
    padding: 2px; /* Padding */
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Select the screen were&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;you want to display the application&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #ffffff; /* White background */
    color: #333333; /* Dark gray text */
    border: 2px solid #719fee; /* Blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Padding inside the line edit */
    font-size: 16px; /* Font size */
}

QLineEdit:hover {
    background-color: #f0f8ff; /* Light blue background on hover */
    border: 2px solid #5b8bcc; /* Darker blue border on hover */
}

QLineEdit:focus {
    background-color: #ffffff; /* White background when focused */
    border: 2px solid #3a6fab; /* Even darker blue border when focused */
    outline: none; /* Remove the default focus outline */
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;

QLabel {
    color: #333333; /* Dark gray text */
    font-size: 14px; /* Font size */
    padding: 2px; /* Padding */
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;|&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_code">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #ffffff; /* White background */
    color: #333333; /* Dark gray text */
    border: 2px solid #719fee; /* Blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Padding inside the line edit */
    font-size: 16px; /* Font size */
}

QLineEdit:hover {
    background-color: #f0f8ff; /* Light blue background on hover */
    border: 2px solid #5b8bcc; /* Darker blue border on hover */
}

QLineEdit:focus {
    background-color: #ffffff; /* White background when focused */
    border: 2px solid #3a6fab; /* Even darker blue border when focused */
    outline: none; /* Remove the default focus outline */
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
QLabel {
    color: #333333; /* Dark gray text */
    font-size: 14px; /* Font size */
    padding: 2px; /* Padding */
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Code&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;|&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_testScreen">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
            background-color: #A9A9A9;
            color: white;
            border: 2px solid #000000;
            border-radius: 10px;
            padding: 5px 15px;
            font-size: 16px;
            font-weight: bold;
            text-align: center;
        }
        QPushButton:hover {
            background-color: #5b8bcc;
            border: 2px solid #3a6fab;
        }
        QPushButton:pressed {
            background-color: #3a6fab;
            border: 2px solid #1f4f8b;
        }</string>
    </property>
    <property name="text">
     <string>Test Screen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Save">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
            background-color: #A9A9A9;
            color: white;
            border: 2px solid #000000;
            border-radius: 10px;
            padding: 5px 15px;
            font-size: 16px;
            font-weight: bold;
            text-align: center;
        }
        QPushButton:hover {
            background-color: #5b8bcc;
            border: 2px solid #3a6fab;
        }
        QPushButton:pressed {
            background-color: #3a6fab;
            border: 2px solid #1f4f8b;
        }</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Inscription">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
            background-color: #A9A9A9;
            color: white;
            border: 2px solid #000000;
            border-radius: 10px;
            padding: 5px 15px;
            font-size: 16px;
            font-weight: bold;
            text-align: center;
        }
        QPushButton:hover {
            background-color: #5b8bcc;
            border: 2px solid #3a6fab;
        }
        QPushButton:pressed {
            background-color: #3a6fab;
            border: 2px solid #1f4f8b;
        }</string>
    </property>
    <property name="text">
     <string>Inscription</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Close">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>540</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
            background-color: #A9A9A9;
            color: white;
            border: 2px solid #000000;
            border-radius: 10px;
            padding: 5px 15px;
            font-size: 16px;
            font-weight: bold;
            text-align: center;
        }
        QPushButton:hover {
            background-color: #5b8bcc;
            border: 2px solid #3a6fab;
        }
        QPushButton:pressed {
            background-color: #3a6fab;
            border: 2px solid #1f4f8b;
        }</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>821</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>811</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>811</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>160</y>
      <width>811</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>811</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>180</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
QLabel {
    color: #333333; /* Dark gray text */
    font-size: 14px; /* Font size */
    padding: 2px; /* Padding */
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Test your screen&lt;br/&gt;before saving&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
